streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -919,13 +904,13 @@ private long adjustRecordsProcessedBeforeCommit(final long prevRecordsProcessedB
         if (processLatency > 0 && processLatency > commitTime) {
             // push down
             recordsProcessedBeforeCommit = Math.max(1, (commitTime * totalProcessed) / processLatency);
-            log.debug("{} processing latency {} > commit time {} for {} records. Adjusting down recordsProcessedBeforeCommit={}",
-                logPrefix, processLatency, commitTime, totalProcessed, recordsProcessedBeforeCommit);
+            log.debug("processing latency {} > commit time {} for {} records. Adjusting down recordsProcessedBeforeCommit={}",
+                    processLatency, commitTime, totalProcessed, recordsProcessedBeforeCommit);
```

Comment:
nit: formatting. Ideally params will be on a separate line

Reply:
they are on the separate line, am I missing something ?

Reply:
sorry, i meant one per-line. However, it was already like that so I'm not overly bothered